packages/components/src/tabs/stories/index.story.tsx

```diff
@@ -20,7 +20,8 @@ import Tooltip from '../../tooltip';
 import Icon from '../../icon';
 
 const meta: Meta< typeof Tabs > = {
-	title: 'Components (Experimental)/Tabs',
```

Comment:
In most cases, we can't just move components under the Experimental grouping because there are other administrative things that need to be done.

But in this case, since it's a "private" component, we can do so by just adding a redirect (aeec693c818b879e1606d526a922283ee56fbd8f) so the previous permalink (`http://localhost:50240/?path=/docs/components-experimental-tabs--docs`) redirects to the new one 👍